packs/lacework/dashboards/lacework-anomaly-detections.json

```diff
@@ -0,0 +1,197 @@
+{
+  "name": "Lacework Anomaly Detections",
+  "description": null,
```

Comment:
I am expecting a few schema failures, we are looking into this because I assume you just used the export button, but there are a few quick fixes to get this moving:
* `description` cannot be null, they also have to be defined in the `pages` section (they can be an empty string though)
* the `permissions` field needs to be removed

Reply:
Done!